src/workerd/jsg/jsg.h

```diff
@@ -47,6 +47,7 @@ namespace workerd::jsg {
       ::workerd::jsg::JsgKind::RESOURCE; \
   using jsgSuper = jsgThis; \
   using jsgThis = Type; \
+  inline kj::StringPtr jsgGetName() const { return #Type##_kjc; } \
```

Comment:
@jasnell I'm not sure what to do here. The build fails when I don't include it, but the `Type` here is the name of the isolate wrapper, not the wrapped types, so we get weird results in test cases.